server/server.go

```diff
@@ -313,7 +314,9 @@ func (s *Server) ListenAndServe(leader string) error {
 	s.router.HandleFunc("/diagnostics", s.serveDiagnostics).Methods("GET")
 	s.router.HandleFunc("/raft", s.serveRaftInfo).Methods("GET")
 	s.router.HandleFunc("/db", s.readHandler).Methods("GET")
+	s.router.HandleFunc("/db/{tablename}", s.tableReadHandler).Methods("GET")
 	s.router.HandleFunc("/db", s.writeHandler).Methods("POST")
+	s.router.HandleFunc("/db/ingest", s.tableWriteHandler).Methods("POST")
```

Comment:
How about using `/db/bulk` instead? Both `POST` endpoints allow data to be "ingested".